cloudformation/service/eu-west-2/cognito.yml

```diff
@@ -38,6 +40,62 @@ Resources:
+                                ></path></svg
+                              ><span style="padding-left: 5px;"
+                                >GOV.UK</span
+                              ></span
+                            ></a
+                          >
+                        </div>
+                        <div style="width: 66.66%; padding-left: 15px;">
+                          <a style="color: #ffffff;display: inline-block; margin-bottom: 10px; font-size: 25px; font-weight: 700; text-decoration: none; line-height: 2.5;"
+                            href="/"
+                            id="title_link"
+                            >Fares Data Build Tool</a
+                          >
+                        </div>
+                      </div>
+                    </header>
+                    <div style="font-family: Arial, sans-serif; padding-left: 30px">
+                        <h1>Create account - Fares Data Build Tool</h1>
+                        <p>To create an account to access the Fares Data Build Tool click the link below:</p>
+                        <a href="${RegistrationLink}?key={####}">Link to registration</a>
+                        <p>The link is valid for 72 hours for the following email address email address here</p>
```

Comment:
I've just ran through the process on the console but my email address wasn't populated in the email. I think the line above needs to be dynamic somehow?